File: Frameworks/MetalPetal/MTIFilter.m
//
//  MTIFilter.m
//  Pods
//
//  Created by YuAo on 25/06/2017.
//
//

#import "MTIFilter.h"
#import "MTIVertex.h"
#import "MTIImageRenderingContext.h"
#import "MTIImage.h"
#import "MTIFilterFunctionDescriptor.h"
#import "MTIFilter+Property.h"


NSString * const MTIFilterPassthroughVertexFunctionName = @"passthroughVertexShader";
NSString * const MTIFilterPassthroughFragmentFunctionName = @"passthroughFragmentShader";

@interface MTIFilter ()

@end

@implementation MTIFilter

- (MTIImage *)outputImage {
    return nil;
}

@end

File: Frameworks/MetalPetal/MTIFilter.swift
//
//  MTIFilter.swift
//  Pods
//
//  Created by YuAo on 22/09/2017.
//

import Foundation

extension MTIFilter {
    
    public var parameters: Dictionary<String, Any> {
        return MTIGetParametersDictionaryForFilter(self)
    }
    
}
